src/query/storages/stage/src/read/avro/avro_to_jsonb.rs

```diff
@@ -15,54 +15,126 @@
 use std::collections::BTreeMap;
 
 use apache_avro::types::Value;
+use apache_avro::Schema;
+use databend_common_expression::types::i256;
+use databend_common_expression::types::Decimal;
+use databend_common_expression::types::MAX_DECIMAL128_PRECISION;
+use databend_common_expression::types::MAX_DECIMAL256_PRECISION;
+use num_bigint::BigInt;
 
-pub(super) fn to_jsonb(value: &Value) -> Result<jsonb::Value, String> {
-    let jvalue = match value {
-        Value::Null => jsonb::Value::Null,
-        Value::Boolean(v) => jsonb::Value::Bool(*v),
-        Value::Int(v) => jsonb::Value::from(*v),
-        Value::Long(v) => jsonb::Value::from(*v),
-        Value::Float(v) => jsonb::Value::from(*v),
-        Value::Double(v) => jsonb::Value::from(*v),
-        Value::String(v) => jsonb::Value::from(v.as_str()),
-        Value::Enum(_, v) => jsonb::Value::from(v.as_str()),
-        Value::Uuid(v) => jsonb::Value::from(v.to_string()),
-        Value::Union(_, v) => to_jsonb(v)?,
-        Value::Array(v) => {
+pub(super) fn to_jsonb<'a>(value: &'a Value, schema: &Schema) -> Result<jsonb::Value<'a>, String> {
```

Comment:
better to add some unit test for this function to ensure that the value can be correctly converted to jsonb format value and can be converted back to the original value.

Reply:
I will add some test later, but not all type can be converted back.

Reply:
add tests mainly for decimal,
for the other types, unit test code are almost the same as the code being tests.
the will be tested in logic test after cast functions are add https://github.com/databendlabs/databend/issues/17940